docs-markdown/src/helper/triple-colon-extensions.ts

```diff
@@ -0,0 +1,109 @@
+import { CancellationToken, CompletionContext, CompletionItem, CompletionItemKind, CompletionItemProvider, MarkdownString, Position, Range, SnippetString, TextDocument } from "vscode";
+import { matchAll } from "./common";
+
+/* Triple colon extensions for completion provider
+* used to quickly insert the triple colon syntax into markdown
+*/
+export interface ITripleColonExtension {
+    readonly content: string;
+    readonly extension: string;
+}
+
+export type TripleColonExtensions = ITripleColonExtension[];
+
+const tripleColonExtensions: TripleColonExtensions = [
+    {
+        content: `column:::
+    {content}
+:::column-end:::`,
+        extension: "Column :::column:::",
+    },
+    {
+        content: `image type="content" source="{source}" alt-text="{alt-text}":::`,
+        extension: "Image :::image:::",
+    },
+    {
+        content: `moniker range="{range}"
```

Comment:
When I use moniker, the indenting is off:
![image](https://user-images.githubusercontent.com/1048330/81352794-3f443300-907c-11ea-8bc1-5a371c119e76.png)

Reply:
Fixed the indent on moniker option.
![image](https://user-images.githubusercontent.com/51799435/81353545-36ecf780-907e-11ea-8f2b-ffd5f69d3e3d.png)